index.js

```diff
@@ -27,7 +27,9 @@ module.exports = async (patterns, {force, dryRun, ...options} = {}) => {
 		...options
 	};
 
-	const files = await globby(patterns, options);
+	const files = (await globby(patterns, options))
+		.sort((a, b) => a.localeCompare(b))
```

Comment:
```suggestion
		.sort((a, b) => b.localeCompare(a))
```

Then you can drop the `.reverse()`.

Reply:
Good idea, I missed that one! Changed.